src/test/java/org/folio/rest/impl/ReferenceAndSampleDataIT.java

```diff
@@ -6,56 +6,33 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 
-import org.folio.postgres.testing.PostgresTesterContainer;
-import org.folio.rest.persist.PostgresClient;
-import org.folio.rest.tools.utils.NetworkUtils;
-import org.folio.support.VertxModule;
+import org.folio.moduserstest.AbstractRestTest;
 import org.folio.support.http.AddressTypesClient;
-import org.folio.support.http.FakeTokenGenerator;
 import org.folio.support.http.GroupsClient;
-import org.folio.support.http.OkapiHeaders;
-import org.folio.support.http.OkapiUrl;
 import org.folio.support.http.UsersClient;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
-import io.vertx.core.Vertx;
 import io.vertx.junit5.Timeout;
 import io.vertx.junit5.VertxExtension;
-import io.vertx.junit5.VertxTestContext;
-import lombok.SneakyThrows;
 
 // Loading the reference and sample data can take a long time
 @Timeout(value = 30, timeUnit = SECONDS)
 @ExtendWith(VertxExtension.class)
-class ReferenceAndSampleDataIT {
+class ReferenceAndSampleDataIT extends AbstractRestTest {
```

Comment:
Should messages to be sent to Kafka for users created when sample data is loaded?

Reply:
As for me, it is correct, because we should not differ sample data users and real users, even if mod-consortia will be disabled - in this case events for sample data users will stay in the topic until expiration period finishes, but it is OK

Reply:
When a user, any user is created in mod-users, the appropriate event should be sent. We should not differentiate between sample and non-sample users.

Reply:
If this is important behaviour, maybe that could be checked in these tests